glusterd2/store/liveness.go

```diff
@@ -39,19 +41,26 @@ func (s *GDStore) IsNodeAlive(nodeID interface{}) bool {
 	key := LivenessKeyPrefix + keySuffix
 	resp, err := s.Get(ctx, key)
 	if err != nil {
-		return false
+		return 0, false
 	}
 
-	return resp.Count == 1
+	if resp.Count == 1 {
+		pid := string(resp.Kvs[0].Value)
```

Comment:
The variable name `id` and `pid` indicate that they are two different things. You can simply do this instead:

```go
pid, err := strconv.Atoi(string(resp.Kvs[0].Value))
```

Reply:
done